test/strategy/auth0/oauth_test.exs

```diff
@@ -1,12 +1,13 @@
 defmodule Ueberauth.Strategy.Auth0.OAuthTest do
   use ExUnit.Case
 
-  import Ueberauth.Strategy.Auth0.OAuth, only: [client: 0, client: 1]
+  import Ueberauth.Strategy.Auth0.OAuth, only: [client: 1, client: 2]
 
   @test_domain "example-app.auth0.com"
 
   setup do
-    {:ok, %{client: client()}}
+    conn = %Plug.Conn{}
+    {:ok, %{client: client(conn)}}
```

Comment:
Is it possible to keep a test with `client/0` that doesn't need a `%Plug.Conn{}` as first argument ?

That way, the `config_from` would be opt-in for those that need it but not required everywhere.

Reply:
Yes, it's possible 😀 . I changed it so`conn` it can be passed through `opts`.